.github/CODEOWNERS

```diff
@@ -1,7 +1,7 @@
 # Each line is a file pattern followed by one or more owners.
 
 # global
-* @hypertrace/config-services-owners
+* @hypertrace/query-layer-owners
```

Comment:
This service can really go either way, it's both query and config - I'm fine either way, though.